scripts/run-docker-validator-network.sh

```diff
@@ -380,7 +382,7 @@ if [[ $COMMAND == *"run-fullnode"* ]]; then
     echo -e "\t 'screen -r -S celo-fullnode'\n"
 fi
 
-if [[ $COMMAND == *"print-env"* ]]; then
```

Comment:
What is the meaning of `|| [ 1 ]]`?

Reply:
The intention was to force the printing of the environment variables. But maybe was not clear enough. I made this up adding a variable for improve the readability of this.